trigger/host_spawn.go

```diff
@@ -185,6 +185,12 @@ func (t *spawnHostStateChangeTriggers) spawnHostStateChangeOutcome(sub *event.Su
 		//   running.
 		return nil, nil
 	}
+	if t.data.Successful && t.data.Source == string(evergreen.ModifySpawnHostSleepSchedule) || t.data.Source == string(evergreen.ModifySpawnHostSleepScript) {
```

Comment:
Do we only want `Successful` to be considered for sleep schedule modifications and not for sleep scpript modifications?

Reply:
Hmm, I debated a little bit whether or not the sleep script should really suppress notifications and thinking more on it, I think it would actually be preferable for now to let users know when their script is acting since it might stop the host when the user isn't expecting it. Unlike a regular sleep schedule, the script acts based on checking for specific activities on the host, so users might not be aware of when their host shuts down if they choose to use it.

We can consider readjusting the notifications later on if it turns out that users of the script request to suppress these notifications.